lib/v2/datasource/local/util/seeds_esr.dart

```diff
@@ -13,6 +15,23 @@ class SeedsESR {
   Future<void> resolve({String? account}) async {
     actions = await manager.fetchActions(account: account);
   }
+
+  Result processResolvedRequest() {
```

Comment:
this doesnt seem like something that should be in a utils class. 

whats this method doing? 
How is it used? 

is this a use case?

Reply:
This was put here from the previous code, since it is used in two places

In the interest of not creating a possibility for this to break, I would leave it like that for the time being.

There's ways to refactor this functionality so it makes more sense

What's happening here is that we convert the parsed and resolved (!) ESR object into a data object representing an action. 

This was like this in the old code except only in 1 place. 

The object should be called ActionData

But I don't know if I would even do that if revisiting this code - it seems to make little sense. Why not pass the ESR object around? Or an Action object? 

I think the correct fix would cause more changes and more possibilities of breaking the error handling etc. So would leave as is.

Reply:
I could put a comment on this

Reply:
Added a big comment - getting more convinced that this whole code structure has to be removed and done an entirely different way. In a different PR. I added a TODO.